database/storage.go

```diff
@@ -137,9 +132,55 @@ func (s *Storage) rows(table string) (*sql.Rows, error) {
 }
 
 func (s *Storage) nRows(table string, n string) (*sql.Rows, error) {
-	nRows, err := s.conn.Query(fmt.Sprintf("SELECT * FROM `%s` LIMIT %s", table, n))
+	column, err := s.primaryColumn(table)
+	if err != nil {
+		return nil, err
+	}
+	q := fmt.Sprintf("SELECT * FROM `%s` ORDER BY %s DESC LIMIT %s", table, column, n)
```

Comment:
@rafaeljesus @mereba using sprintf will make this vulnerable to SQL Injection, please use prepared statements / parametrized queries.
seems like Query() accepts parameters and will do the "concatenation" for you, https://golang.org/pkg/database/sql/#DB.Query

Reply:
If I see the scope of this project correctly, you need to have full db access anyway 😅 But in general @jhuliano is right to use parameterized queries. Good resources for Golang + SQL:
http://go-database-sql.org/
There is also a nice wrapper that adds some "more comfy" helpers http://jmoiron.github.io/sqlx/

Reply:
I don't think the scope of the project should matter though, packages/projects evolve and code gets reused and this might lead to a pretty bad security vulnerability.

Reply:
@jhuliano I did not disagree with you 😉

Reply:
sqlx FTW

Reply:
Thanks guys! Made [the changes](https://github.com/hellofresh/klepto/pull/29/commits/308f19f21e3f0f7c2ccb151d874dcc181e543df8) where possible